### ISSUE TITLE
required_text property of Form Builder Class does not generate a string in the form

### ISSUE THREAD

**Issue by dgvirtual** [daylightstudio/FUEL-CMS#468]
I am using FuelCMS 1.4.2. When I include property "required" in the field config, e.g. like this: 

`$fields['name'] = array('type' => '','required' => TRUE);`

the Form Builder class does produce the asterix (*) after the form (or another character, as specified by the "required_indicator" property), but the asterix is not followed by a "required_text" string. Likewise, if I override the default "required_indicator" property, e.g. like this: 

`$this->form_builder->show_required = '<span class="required">{required_indicator}</span> these fields are super required'; `

the text does not appear in the form either.

**Comment by daylightstudio** [comment]
How are you implementing this code? In a custom controller?

**Comment by dgvirtual** [comment]
Yes, the controller that I posted here: https://forum.getfuelcms.com/discussion/comment/10641 (you can see I turned it off, as it did not work).

**Comment by daylightstudio** [comment]
The `show_required` is meant for the small bottom (or top) text to indicate required fields must be filled out and is not the text displayed next to the form field's label which will only be the `required_indicator`.
`show_required`: which can be TRUE, FALSE, or "top" to display the required text verbiage at the bottom or top (default is bottom)
`required_indicator`: the asterisk to display next to the form label
`required_text`: the text to show at the bottom (or top) of the form indicating required fields in the form (e.g. The fields indicated with a "*" are required)

**Comment by dgvirtual** [comment]
Yes, that is what I meant: that "the small bottom (or top) text to indicate required fields must be filled out" is not displayed at the bottom (or top) of the form - only the lonely asterix is displayed. I am not referring to "the text displayed next to the form field's label". As the lonely asterix does not look good, I turned it of with  `$this->form_builder->show_required = FALSE; `, but that is not a solution, just a workaround for me.

**Comment by dgvirtual** [comment]
Ok, I have tried to load it right before the form_builder and it worked. 
now I get `* required fields` text after the form. And I see I have made a mistake in trying to convince show_required to play the part of required_text... Sorry. It all works now, and my error report is not valid. 

However, how is the form_builder to be used by default, if not in a controller? The documentation at docs.getfuelcms.com/general/forms does not mention it and the examples look very much "controller - like".

**Comment by daylightstudio** [comment]
You are right, it should probably be included in the docs if just used in a controller.
